types/eslint/index.d.ts

```diff
@@ -1335,8 +1335,12 @@ export class ESLint {
 export namespace ESLint {
     type ConfigData<Rules extends Linter.RulesRecord = Linter.RulesRecord> = Omit<Linter.Config<Rules>, "$schema">;
 
+    interface Globals {
```

Comment:
How about `EnvironmentGlobals`?

Reply:
I don't think that's accurate.

I think the `interface Environment` was only named because it's used in the `environments` property in `Plugin`:
https://github.com/privatenumber/DefinitelyTyped/blob/f1b0d462348f19165b4719d04bc6662bfd5dfbe1/types/eslint/index.d.ts#L1362

But in `FlatConfig`, it's used in `languageOptions`:
https://github.com/privatenumber/DefinitelyTyped/blob/f1b0d462348f19165b4719d04bc6662bfd5dfbe1/types/eslint/index.d.ts#L1228


In the docs, ["Environment"](https://eslint.org/docs/latest/use/configure/language-options#specifying-environments) and ["Globals"](https://eslint.org/docs/latest/use/configure/language-options#specifying-globals) mean different things.

Reply:
Fair enough.